### ISSUE TITLE
Failing to open any folder

### ISSUE THREAD

**Issue by coldwaterq** [keybase/client#4168]
When I try to open, my public folder, my private folder, or a shared private folder I have I get Keybase saying it timed out. And Windows says that I have insufficient resources to open the directory.

my log id after only trying to open my private folder is: 5d3b025ffa876867d6d8b01c
my log id after trying to open all three folders is : eb10b4ca3ff74b1d90e3da1c

**Comment by strib** [comment]
Looks like a dup of #3894.  Looks like the reporter of that bug closed it even though we haven't fixed it for sure yet, but we think the next Windows release will behave better.

In the meantime, you can try restarting Keybase and it should go away.  Basically there's a bug where KBFS doesn't think your secret key is unlocked, because we're stuck somewhere waiting for your passphrase (but with no way for you to enter it).

**Comment by coldwaterq** [comment]
Yep, restarting it made it work. If it helps I tried using keybase pgp
decrypt before entering the password so maybe that got it in the bad state.

**Comment by strib** [comment]
^ @patrickxb @maxtaco: maybe interesting info about the passphrase lockup you were looking into in #3894.
